WordPress/Classes/ViewRelated/Blog/QuickStartNavigationSettings.swift

```diff
@@ -1,12 +0,0 @@
-class QuickStartNavigationSettings: NSObject {
-    func updateWith(navigationController: UINavigationController, andViewController viewController: UIViewController) {
-
-        switch viewController {
-        case is BlogListViewController:
```

Comment:
AFAIU, this does nothing in the current code. There is also no part of the current Quick Start that guides you through the site picker - if you have a single site, there is no site picker.

P.S. Quick Start is the probably one of biggest problem in the app in terms of maintainability.